asset_data.py

```diff
@@ -43,24 +56,12 @@ def update_global_data(self):
         )
         global_response.raise_for_status()
         self.cg_global_data = global_response.json()
-        self.crypto_mcap_perc_total = self.cg_global_data["data"]["market_cap_percentage"]
+        if self.cg_global_data["data"]["market_cap_percentage"]:
+            self.crypto_mcap_perc_total = self.cg_global_data["data"]["market_cap_percentage"]
+        if self.cg_global_data["data"]["total_market_cap"]["usd"]:
+            self.crypto_total_mcap = '${:,.2f}'.format(self.cg_global_data["data"]["total_market_cap"]["usd"])
 
-    def format_data(self, asset, asset_ticker):
-        """Formats data into a more readable format. Adds commas/punctuation and rounds to two decimal places."""
-        asset_dict = {"price": '${:,.2f}'.format(self.cg_asset_data[asset]["usd"]),
-                      "mcap": '${:,.2f}'.format(self.cg_asset_data[asset]["usd_market_cap"]),
-                      "24h_change_percent": f"{round(self.cg_asset_data[asset]["usd_24h_change"], 2)}",
-                      "percent_of_total_mcap": f"{round(self.crypto_mcap_perc_total[asset_ticker], 2)}%"
-                      }
-        return asset_dict
 
-    def daily_icon(self, percent_change):
-        """Takes the 24h % change in price and returns an icon to make it easier
-        to discern divergences between assets/asset classes"""
-        if float(percent_change) > 0:
-            icon = "🔺"
-        elif float(percent_change) < 0:
-            icon = "🔻"
-        else:
-            icon = ""
-        return icon
+if __name__ == '__main__':
+    bob = DataManager()
+    print(bob.get_asset_data("bitcoin"))

```

Comment:
do you need the print or can we remove it?

Reply:
Forgot to remove that 😅